### PR TITLE
Sombreros

### PR DESCRIPTION
**Pull request by RealestEstate**
found a sombrero icon in the codebase without any code so this pr adds it to the autodrobe where its cousin the poncho resides.
[content]

:cl:
 * rscadd: the Autodrobe is now stocked with some sombreros. Olé.

### CODE REVIEW

**Comment by DamianX** [comment]
Which DMI was it in originally?

**Comment by Kurfursten** [comment]
>called sombrero
>it's "been here all along"

Blizzard viral marketing/ARG?

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -539,3 +539,10 @@
 	item_state = "pharaoh"
 	wizard_garb = TRUE
 	body_parts_covered = FULL_HEAD|HEAD|EARS
+
+/obj/item/clothing/head/sombrero
+	name = "sombrero"
+	desc = "Meanwhile in Space Mexico"
+	icon_state = "sombrero"
+	item_state = "sombero"
```
Comment (DamianX) [inline code/modules/clothing/head/misc.dm]: typo?

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -539,3 +539,10 @@
 	item_state = "pharaoh"
 	wizard_garb = TRUE
 	body_parts_covered = FULL_HEAD|HEAD|EARS
+
+/obj/item/clothing/head/sombrero
+	name = "sombrero"
+	desc = "Meanwhile in Space Mexico"
```
Comment (jknpj) [inline code/modules/clothing/head/misc.dm]: Neo Space Mexico

File: code/modules/clothing/head/misc.dm (hunk 1 above)
Comment (RealestEstate) [reply code/modules/clothing/head/misc.dm]: typo

File: code/modules/clothing/head/misc.dm (hunk 2 above)
Comment (RealestEstate) [reply code/modules/clothing/head/misc.dm]: thanks 👌

**Comment by RealestEstate** [comment]
@DamianX in icon/mob/head.dmi.
both this and the poncho had sprites and no code, but I implemented the poncho a bit ago

File: code/modules/clothing/head/misc.dm (hunk 2 above)
Comment (Intigracy) [reply code/modules/clothing/head/misc.dm]: add .

**Comment by Intigracy** [comment]
arriba arriba andale arriba

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -539,3 +539,10 @@
 	item_state = "pharaoh"
 	wizard_garb = TRUE
 	body_parts_covered = FULL_HEAD|HEAD|EARS
+
+/obj/item/clothing/head/sombrero
+	name = "sombrero"
+	desc = "Meanwhile in Neo Space Mexico."
```
Comment (jknpj) [inline code/modules/clothing/head/misc.dm:545]: image

**Pull request merged**